docs/RECIPES.md

```diff
@@ -0,0 +1,136 @@
+# Preflight Usage Examples
+
+Below are detailed examples on how to leverage `preflight` in various
+environments.
+
+These examples are shown using the Operator policy against an operator bundle
+(e.g. `preflight check operator <bundle`>), but the same concepts can be applied
+to the Container policy (give or take a few configuration changes).
+
+You will also need an index image containing your bundle for each of these approaches.
+See [DOCS](BUILDING_AN_INDEX.md)
+
+## As a Binary on Your Workstation
+
+To run `preflight` on your workstation, you'll first need to download and
+[install](../README.md#Installation)
+the binary to your path.
+
+You will also need:
+
+- Your bundle image published to a container registry,
+- An Index Image containing your operator's new bundle published to a container
+  registry.
+- A Kubeconfig for a user with cluster-admin privileges to an OpenShift 4.5+
```

Comment:
Is this true? Does it have to be cluster-admin? My testing has shown that to not be the case.

Reply:
I'll address this in the other comment related to the same topic 👍